test/e2e/taints_test.go

```diff
@@ -226,7 +231,7 @@ var _ = framework.KubeDescribe("NoExecuteTaintManager [Serial]", func() {
 	// 4. See if pod will get evicted after toleration time runs out
 	It("eventually evict pod with finite tolerations from tainted nodes", func() {
 		podName := "taint-eviction-3"
-		pod := createPodForTaintsTest(true, 5, podName, ns)
+		pod := createPodForTaintsTest(true, KubeletPodDeletionDelaySeconds+2*AdditionalWaitPerDeleteSeconds, podName, ns)
```

Comment:
I need to create a Pod with a toleration that will be longer than normal time we wait for deletion (deletion delay + slack), to make sure it wasn't evicted after waiting (deletion delay + slack).